crest/Assets/Crest/Crest/Shaders/Macros.hlsl

```diff
@@ -0,0 +1,15 @@
+// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
+// MACROS - Macros to help with cross pipeline development
+
+#ifndef CREST_MACROS_H
+#define CREST_MACROS
+
+#ifndef CBUFFER_START
```

Comment:
So did you need to add a stub for this in the end - did the macro not work in 2018? this is a good solution and looks good, im just asking out of curiosity

Reply:
Actually... I think I have done the wrong thing here. The macros aren't defined, but I didn't check if the stub is required. I've made an assumption. Also, there is a typo which I have fixed. I can copy the code from the PPv2 stack which defines these very macros for compute.

Reply:
I have implemented the stub from PPv2. Tested working fine on Metal. Thanks!